catalog/spatial/geocoding/spatial-geocoding-extract/src/main/java/org/codice/ddf/spatial/geocoding/extract/GeoNamesFileExtractor.java

```diff
@@ -111,7 +112,7 @@ public void pushGeoEntriesToExtractionCallback(final String resource,
             for (String line; (line = reader.readLine()) != null;) {
                 extractionCallback.extracted(extractGeoEntry(line));
                 bytesRead += line.getBytes().length;
-                extractionCallback.updateProgress((int) ((bytesRead / fileSize) * 100));
+                extractionCallback.updateProgress((int) (50 + (bytesRead / fileSize) * 50));
```

Comment:
0% - 50% is the status of obtaining the file to extract (https://github.com/codice/ddf/blob/master/catalog/spatial/geocoding/spatial-geocoding-extract/src/main/java/org/codice/ddf/spatial/geocoding/extract/GeoNamesFileExtractor.java#L215)
51% - 100% is the status of the extracting and indexing the file (https://github.com/bdeining/ddf/blob/DDF-1655/catalog/spatial/geocoding/spatial-geocoding-extract/src/main/java/org/codice/ddf/spatial/geocoding/extract/GeoNamesFileExtractor.java#L115).

The current implementation separates the progress of obtaining the file (download or ingest) with the extraction and indexing, which can be confusing.  This was changed to make the progress of the entire process 0-100% instead of resetting the progress during the updateIndex() invocation.